### ISSUE TITLE
servicePath is not well manage

### ISSUE THREAD

**Issue by pgodot01** [vfarcic/docker-flow-monitor#12]
Thanks for this stack.

I tried to configure demo micro service with following configuration in my docker stack:

    deploy:
      labels:
        - com.df.notify=true
        - com.df.distribute=true
        - com.df.servicePath=/demo
        - com.df.port=8080
 
In my swarm-listener logs, I got:
monitor_swarm-listener.1.yykqznc875td@manager    | 2017/10/30 12:00:18 Sending service created notification to http://monitor:8080/v1/docker-flow-monitor/reconfigure?distribute=true&port=8080&replicas=1&serviceName=demo_main&servicePath=%2Fdemo

But in my monitor_monitor logs, I got:
monitor_monitor.1.vcn8myrn9igg@manager    | 2017/10/30 12:00:18 Processing /v1/docker-flow-monitor/reconfigure?distribute=true&port=8080&replicas=1&serviceName=demo_main&servicePath=**%!F(MISSING)**demo
monitor_monitor.1.vcn8myrn9igg@manager    | GLOBAL_SCRAPE_INTERVAL=10s

I have this strange MISSING string. Any Idea why ?

Thanks
Phil

**Comment by pgodot01** [comment]
Does go_demo expose metrics on /demo URI or on /metrics ?

**Comment by vfarcic** [comment]
It exposes them on `/metrics` (https://github.com/vfarcic/go-demo/blob/master/main.go#L73).

**Comment by pgodot01** [comment]
Is it possible to configure the metrics_path of prometheus scrap in the monitored container? My application is exposing its metrics at /prometheus.

**Comment by vfarcic** [comment]
It's added in the [release 17.10.30-22](https://github.com/vfarcic/docker-flow-monitor/releases/tag/17.10.30-22). Can you please try it out and let me know whether it works as expected?

**Comment by pgodot01** [comment]
Yes with pleasure. I checked your commit. It seems what I need. Have you implemented a verb version in your api to check if my docker image is up to date?

**Comment by pgodot01** [comment]
Well done it's working like a charm.
